AsyncSwift/Extensions/SafariView.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  SafariView.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/11/04.
+//
+
+import Foundation
+import SwiftUI
+import SafariServices
```

Comment:
```suggestion
import SafariServices
import SwiftUI
```

이번에 수정된 파일들에서 전체적으로 Foundation을 삭제할 필요가 있어보입니다

Reply:
[refactor : removed 'import Foundation'](https://github.com/Async-Swift/AsyncSwift/pull/60/commits/b4c445ca4accbe5c6b1ae358006bedabb4edf21f)

수정했습니다
감사합니다